src/Bonsai.ML.Torch/LinearAlgebra/Norm.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch.LinearAlgebra
+{
+    /// <summary>
+    /// Computes a matrix norm.
+    /// </summary>
+    [Combinator]
+    [Description("Computes a matrix norm.")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+    public class MatrixNorm
```

Comment:
```suggestion
    public class Norm
```

For consistency with TorchSharp I guess this would be called just `norm` instead of `MatrixNorm`?